modules/enet/doc_classes/ENetConnection.xml

```diff
@@ -61,7 +61,9 @@
 			<param index="2" name="in_bandwidth" type="int" default="0" />
 			<param index="3" name="out_bandwidth" type="int" default="0" />
 			<description>
-				Create an ENetHost that will allow up to [param max_peers] connected peers, each allocating up to [param max_channels] channels, optionally limiting bandwidth to [param in_bandwidth] and [param out_bandwidth].
+				Creates an ENetHost that allows up to [param max_peers] connected peers, each allocating up to [param max_channels] channels, optionally limiting bandwidth to [param in_bandwidth] and [param out_bandwidth] (if greater than zero).
+				This method binds a randomly numbered UDP port on the host machine at the [i]unspecified[/i] address. Use [method create_host_bound] to specify the address and port.
```

Comment:
```suggestion
				This method binds a random available dynamic UDP port on the host machine at the [i]unspecified[/i] address. Use [method create_host_bound] to specify the address and port.
```
Not sure if this distinction matters, but it is guaruanteed to be a currently unused port in the dynamic/private/ephemeral port range (49152 to 65535).

Reply:
Woow nice catch! I'll apply it.